scripts/compress.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
```

Comment:
Why not adding this script to the Chrome Extension Utils repo ? Could be useful for others (especially if we plan to publish/improve it) :)